app/src/main/java/com/quran/labs/androidquran/service/AudioService.java

```diff
@@ -1081,14 +1107,17 @@ private void setUpAsForeground() {
           .setContentTitle(getString(R.string.app_name))
           .setContentIntent(pi)
           .setVisibility(NotificationCompat.VISIBILITY_PUBLIC)
-          .addAction(R.drawable.ic_previous, getString(R.string.previous), previousIntent)
-          .addAction(R.drawable.ic_pause, getString(R.string.pause), pauseIntent)
-          .addAction(R.drawable.ic_next, getString(R.string.next), nextIntent)
-          .setLargeIcon(icon)
-          .setShowWhen(false)
-          .setStyle(new NotificationCompat.MediaStyle()
-              .setShowActionsInCompactView(new int[]{0, 1, 2})
-              .setMediaSession(mMediaSession.getSessionToken()));
+          .addAction(R.drawable.ic_previous,
+              emptyTitles ? "" : getString(R.string.previous), previousIntent)
```

Comment:
see the screenshots on https://github.com/quran/quran_android/pull/610 - basically, when there is no large notification icon, if we still set `MediaStyle`, it will make the small icon stretch to fit and looks really bad - consequently, that logic is to behave the old way (buttons without titles) when we can't load the large image (and we don't `setStyle` and we also don't `setLargeIcon` in those cases).